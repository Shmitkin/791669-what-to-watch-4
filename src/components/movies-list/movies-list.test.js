import React from "react";
import renderer from "react-test-renderer";
import MoviesList from "./movies-list.jsx";

const testState = [
  {
    title: `First Film`,
    preview: `img/first-film.jpg`
  },
  {
    title: `Second Film`,
    preview: `img/second-film.jpg`
  },
  {
    title: `Third Film`,
    preview: `img/third-film.jpg`
  },
  {
    title: `Fourth Film`,
    preview: `img/fourth-film.jpg`
  }
];

it(`Should MoviesList render correctly`, () => {
  const tree = renderer
    .create(
        <MoviesList
          movies = {testState}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
