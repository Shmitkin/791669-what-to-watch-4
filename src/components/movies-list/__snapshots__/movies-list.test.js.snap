// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MoviesList render correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="First Film"
        height="175"
        src="img/first-film.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        First Film
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Second Film"
        height="175"
        src="img/second-film.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Second Film
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Third Film"
        height="175"
        src="img/third-film.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Third Film
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fourth Film"
        height="175"
        src="img/fourth-film.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Fourth Film
      </a>
    </h3>
  </article>
</div>
`;
